### PR TITLE
feat(nimbus): Update serializers for localized experiments

### PR DESCRIPTION
**Pull request by brennie**
Because

- we want to support localized experiments in Nimbus

this commit

- updates our serializers to support localized experiments.

Co-authored-by: Jason Prickett <jprickett@mozilla.com>

### CODE REVIEW

File: experimenter/experimenter/experiments/api/v5/serializers.py [hunk 1]
```diff
@@ -1566,6 +1632,125 @@ def _validate_bucket_duplicates(self, data):
...
+                        missing_ids.add(sub_id)
+                        return None
+
+                    return substitutions[sub_id]
+                else:
+                    return {k: substitute(v) for k, v in value.items()}
+
+            return value
+
+        subbed = substitute(feature_value)
```
Comment (jaredlockhart) [inline experimenter/experimenter/experiments/api/v5/serializers.py:1739]: Some fun recursion and closures here!  Someone's been reading their SICP ;D

**Review by jaredlockhart — APPROVED** [review]
This all looks fantastic.  Very well laid out, very readable, very well tested.  A+ 10/10 would approve again 🎉 🎉 🎉 🎉

**Comment by jaredlockhart** [comment]
Let's block this on https://github.com/mozilla/experimenter/pull/8777

**Review by yashikakhurana — APPROVED** [review]
@brennie thank you  everything looks 💯

**Pull request merged**